ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -106,26 +107,46 @@
                                             update-type)]))))
 
 (defn- get-provider-concept-ids
-  "Returns a list of collection concept ids for a given provider-id"
+  "Returns a list of collection concept ids for a given provider-id.
+   Throws exception if non are returned."
   [context provider-id]
   (let [concepts (mdb/find-concepts context 
                                     {:provider-id provider-id} 
                                     :collection)]
-    (when (seq concepts)
-      (distinct (map :concept-id concepts)))))
+    (if (seq concepts)
+      (distinct (map :concept-id concepts))
+      (errors/throw-service-errors
+        :bad-request
+        ["No concept-ids are associated with the provider-id."]))))
+
+(defn- validate-concept-ids
+  "Throws exception if there exists invalid
+   concept-ids."
+  [concept-ids]
+  (when-not (= ["ALL"] (map string/upper-case concept-ids))
+    (let [err-msgs 
+           (for [id concept-ids
+                 :let [err-msg (concepts/concept-id-validation id)
+                       err-msg (if err-msg
+                                 err-msg
+                                 (when-not (string/starts-with? id "C")
+                                   [(str "Concept-id [" id 
+                                         "] is not a valid collection concept id, "
+                                         "must start with C")]))]
+                 :when err-msg]
+             err-msg)]
+      (when (seq err-msgs)
+        (errors/throw-service-errors
+          :bad-request
+          [(string/join ", " err-msgs)])))))
```

Comment:
We should use `;` as separate as there is `,` in some of the error messages.

Reply:
Done.